packaging/flatpak/collector.py

```diff
@@ -0,0 +1,33 @@
+import json
+import yaml
+
+with open('mirage.flatpak.base.yaml') as f:
```

Comment:
Double quotes are preferred (applies to this entire script) to match the coding style of src/backend

Reply:
OK